### PR TITLE
Creating Leads and converting

### PR DESCRIPTION
**Pull request by anil5375**
Hi Sudhir,

Please review the PR and merge into Epic

### CODE REVIEW

File: src/test/java/pageobjects/SalesforceObjects/LeadsPage.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public static By saveButton=By.xpath("//button[@name='SaveEdit']");
+    public static By openNotContacted=By.xpath("//a[@title='Open - Not Contacted']");
+    public static By converted=By.xpath("//div[@class='slds-path__scroller_inner scroller']//li//a[@title='Converted']//span[2]");
+    public static By selectConvertedStatusButton=By.xpath("//div/div/button[@data-aura-class='uiButton']/span");
+    public static By convertButton=By.xpath("//div[@class='modal-footer slds-modal__footer']/span//button");
+    public static By convertStatusMessage=By.xpath("//div[@class='title']//h2");
+    public static By gotoLeadsButton=By.xpath("//div[@class='modal-footer slds-modal__footer']/button[2]");
+
+    public static By existingAccountSearch=By.xpath("(//div[@class='header']/span[@class='slds-radio'])[1]");
+   // public static By existingContactSearch=By.xpath("(//div[@class='header']/span[@class='slds-radio'])[2]");
```
Comment (snaphray) [inline src/test/java/pageobjects/SalesforceObjects/LeadsPage.java]: Please remove the commented lines if they are not required

File: src/test/java/SalesforceModules/Leads.java [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+import java.util.HashMap;
+
+import static org.testng.Assert.assertEquals;
+
+
+public class Leads {
+    public static HashMap<String, String> dataMap = new HashMap<String, String>();
+
+    public static void ClickOnHamburgerMenu(String logStep) throws InterruptedException {
```
Comment (snaphray) [inline src/test/java/SalesforceModules/Leads.java]: Please use consistent naming conventions for functions name, change the 'ClickOnHamburgerMenu' to 'clickOnHamburgerMenu' and 'EnterText' to 'enterText'

**Review by snaphray — CHANGES_REQUESTED** [review]
Please check comments

File: src/test/java/pageobjects/SalesforceObjects/LeadsPage.java (hunk 1 above)
Comment (anil5375) [reply src/test/java/pageobjects/SalesforceObjects/LeadsPage.java]: Removed commented line

File: src/test/java/SalesforceModules/Leads.java (hunk 2 above)
Comment (anil5375) [reply src/test/java/SalesforceModules/Leads.java]: updated method names

**Review by snaphray — APPROVED** [review]
code looks fine

**Pull request merged**